docs/docs/policies/index.md

```diff
@@ -0,0 +1,66 @@
+# Getting Started with Policies
+
+Tracee's policies are how you describe what tracee should collect and what actions it should take.
+
+For example, we would like to collect all `sched_process_exec` events from the whole host:
+
+```
+name: get started policy
+description: This policy is a get started example
+scope:
+  - global
+defaultAction: log
+rules:
+  - event: sched_process_exec
+```
+
+Here, we create a basic policy with the scope `global` (for the whole host) and the action of logging the events that match the policy.
+
+We can execute the policy and see its result:
+```
+tracee -p get_started_policy.yaml
+```
+
+Once we execute, we notice we also want to collect the event `sched_process_exit`, so we can change the policy accordingly:
```

Comment:
hmm, here is some information missing, why do we notice that? maybe if you just want to also add that event, we could express it differently, otherwise, people will ask what you know that they don't.
"Once we execute the policy, we decide that we also want to collect the event `sched_process_exit`. We can change the policy to collect the event `sched_process_exit`:"